packages/templates/src/test/raw.js

```diff
@@ -165,6 +165,26 @@ describe('raw template files', () => {
     }
   );
 
+  it.each(templates)(
+    '%s template should only contain videos have isMuted attribute',
+    async (template) => {
+      const { default: templateData } = await import(
+        /* webpackChunkName: "chunk-web-stories-template-[index]" */ `../raw/${template}`
+      );
+
+      for (const { elements } of templateData.pages) {
+        for (const element of elements) {
+          if (element?.type !== 'video' || !element?.resource?.src) {
+            continue;
+          }
+
+          expect(element?.resource?.isMuted).not.toBeUndefined();
+          expect(typeof element?.resource?.isMuted === 'boolean').toBeTrue();
```

Comment:
```suggestion
          expect(element?.resource?.isMuted).toBeBoolean();
```

This is possible because we use [`jest-extended`](https://github.com/jest-community/jest-extended#tobeboolean)

Reply:
YES, I couldn't find it.